plugins/portals-list.js

```diff
@@ -134,6 +157,24 @@ window.plugin.portalslist.fields = [
     },
     defaultOrder: -1,
   },
+  { 
+    title: "V/C",
```

Comment:
```suggestion
    title: 'V/C',
```    
https://eslint.org/docs/rules/quotes#single